paxobuilder.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+import argparse
+import argparse
+import requests 
+import zipfile
+
+print("""
+    [+] -----------------------
+    [+] Paxo Builder 0.1
+    [+] 2023 (c) EletrixTime
+    [+] MIT license
+    [+] -----------------------
+    """)
+
+argParser = argparse.ArgumentParser()
+argParser.add_argument("-n", "--build", help="Build (win/linux)")
+args = argParser.parse_args()
+if args.build and args.build == "win":
+    print("[+] Downloading the source code via github...")
+    os.system("git clone https://github.com/paxo-phone/PaxOS-8.git paxos")
+    print("[+] Downloading CMake")
+    r = requests.get("https://github.com/EletrixtimeYT/cmake-portable/releases/download/0.1/cmakeportable.zip", stream=True) 
```

Comment:
I think it would be better to download it from https://cmake.org/ or maybe host it on https://paxo.fr/